app/views/events/index.html.erb

```diff
@@ -12,6 +12,47 @@
   <header class="mb-4 mt-3">
     <div class="container">
       <div class="row justify-content-center">
+      <div class="h-100">
+      <div id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+  <ol class="carousel-indicators">
+    <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="1"></li>
+    <li data-target="#carouselExampleIndicators" data-slide-to="2"></li>
+  </ol>
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img class="d-block w-100" src="https://wallpapercave.com/wp/wp2650488.jpg" alt="First slide">
+      <div class="carousel-caption d-none d-md-block">
+    <h5>"Itt az elso amerika szoveg"</h5>
+    <p>"Elso kicsi amerika"</p>
+  </div>
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="https://wallpapercave.com/wp/wp2650490.jpg" alt="Second slide">
+      <div class="carousel-caption d-none d-md-block">
+      <h5>"Itt a masodik amerika szoveg"</h5>
+      <p>"Masodik kicsi amerika"</p>
+  </div>
+    </div>
+    <div class="carousel-item">
+      <img class="d-block w-100" src="https://cdn.wallpapersafari.com/38/79/pfCxTm.jpg" alt="Third slide">
+      <div class="carousel-caption d-none d-md-block">
+      <h5>Itt a harmadik amerika szoveg</h5>
+      <p>"Harmadik kicsi amerika"</p>
+  </div>
+    </div>
+  </div>
+  <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
+    <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+    <span class="carousel-control-next-icon" aria-hidden="true"></span>
+    <span class="sr-only">Next</span>
+  </a>
+</div>
+</div>
+
```

Comment:
I guess there was a misunderstanding, as there's no need for a carousel on this page.

Reply:
Yes, you are totally right, I just really like the idea of carousels, but as I mentioned before maybe it would be more appropriate on the landing page, but if you don't like it, np, it was just an idea, which we could take also in consideration.